package.json

```diff
@@ -56,13 +56,15 @@
     "jsdom": "^6.5.1",
     "mkdirp": "^0.5.1",
     "mocha": "^2.3.3",
+    "mock-require": "^2.0.1",
     "npm-run-all": "^1.7.0",
     "nyc": "^6.4.4",
     "rimraf": "^2.5.2",
     "webpack": "^1.12.2"
   },
   "dependencies": {
     "asap": "^2.0.3",
+    "can-use-dom": "^0.1.0",
```

Comment:
Nope, using that module seems fine! I'm just cautious about including something if it's trying to do something more complicated.